### DISCUSSION TITLE
[Feature Request] Provide an H5Sselect_is_contiguous API routine for convenience

### DISCUSSION THREAD

**Discussion by jhendersonHDF** [HDFGroup/hdf5#3536] [Ideas]
This API routine would be very easy to add and could save some VOL developers (or other developers) some effort in implementing this functionality.

**Comment by jwsblokland** [comment]
Yes this would be very useful. The simple case is not so difficult to implement yourself but HDF5 allows for complex selections which is not always so straightforward to determine if it is contiguous or not.
